docs/content/Configuration/Connecting-to-the-Database.md

```diff
@@ -192,14 +192,24 @@ CUBEJS_DB_BQ_CREDENTIALS=$(cat /path/to/key-file.json | base64)
 You can learn more about acquiring Google BigQuery credentials
 [here][link-bigquery-getting-started] and [here][link-bigquery-credentials].
 
-You can set the dataset location using the `CUBEJS_DB_BQ_LOCATION` environment variable.
+You can set the dataset location using the `CUBEJS_DB_BQ_LOCATION` environment
+variable.
 
 ```dotenv
 CUBEJS_DB_BQ_LOCATION=us-central1
 ```
 
 You can find more supported regions [here][link-bigquery-regional-locations].
 
+If your pre-aggregations dataset is larger than 100k rows, we **strongly**
```

Comment:
100k of rows is not true for now, because we still didn't ship streaming support for BigQuery. For now, We support download to memory / unload for BigQuery and it depends on allowed memory for the node.js process.

Reply:
Amazing, thank you for the correction